data/cluster-options.yaml

```diff
@@ -26,12 +26,12 @@ CC-Total_Answered:
   warning-level-low: "4"
   worst-level-high: "20"
   worst-level-low: "0"
-ClickToDial2:
-  label: Dial
-  url: http://esx36-ip21-174.gws.genesys.com/sample/sample.html
-CustomAPI:
+CustomLocalAPI:
```

Comment:
It will be preferable to align the section name with the remote one like: SCAPISampleLocal

Reply:
The sections will be renamed 

- LocalSCAPISample
- RemoteSCAPISample